src/Workspaces/Core/Portable/CodeRefactorings/FixAllOccurences/FixAllContextHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeActions;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    internal static class FixAllContextHelper
+    {
+        public static string GetDefaultFixAllTitle(FixAllContext fixAllContext)
+            => GetDefaultFixAllTitle(fixAllContext.Scope, fixAllContext.State.CodeAction, fixAllContext.Document, fixAllContext.Project);
+
+        public static string GetDefaultFixAllTitle(
+            FixAllScope fixAllScope,
+            CodeAction codeAction,
```

Comment:
if we share FixAllScope, and we just pass in the title, then this code can be completely shared i think.